dvc/commands/experiments/queue_worker.py

```diff
@@ -22,4 +24,9 @@ def add_parser(experiments_subparsers, parent_parser):
         add_help=False,
     )
     parser.add_argument("name", help="Celery worker name.")
+    parser.add_argument(
+        "--clean",
+        action="store_true",
+        help="Automatically cleanup celery broker on shutdown.",
```

Comment:
Why do we need this flag and why not enable this automatically?

Reply:
This is a hidden internal plumbing command for launching our celery workers. We only want to automatically garbage collect from one worker (and not every one when we launch multiple workers in parallel with something like `queue start -j`).

From the user perspective this is still automatic (internally we automatically decide which worker to launch with `--clean`)